build/vgg10-radioml/folding_config/ZCU104_folding_config.json

```diff
@@ -1,136 +1,1193 @@
 {
   "Defaults": {},
-  "FMPadding_hls_0": {
-    "SIMD": 2
+  "StreamingFIFO_rtl_0": {
```

Comment:
If you specify the FIFOs in the folding config json, you don't need to set `auto_fifo_depths=True,` in `build.py`. I would do one or the other.

Reply:
Removed auto sizing because this saves time.